pkg/plugin/plugin.go

```diff
@@ -320,7 +379,16 @@ func CmdAdd(args *skel.CmdArgs) error {
 		hostIface, contIface, err = setupVeth(contNetns, args.IfName, mac, netconf.MTU)
 	}
 
-	if err = attachIfaceToBridge(ovsDriver, hostIface.Name, contIface.Name, vlanTagNum, trunks, portType, args.Netns, ovnPort); err != nil {
+	if len(cvlans) > 0 {
```

Comment:
This should not happen in usual code. Setup of vlan ifaces inside the netns is expected to be done by the user. We should whitelist those IDs for them, but they are using it. So this whole block should be dropped and used only for testing.

Reply:
I see, thanks! Fixed